docs/ref/django-admin.txt

```diff
@@ -1793,15 +1793,48 @@ Extra niceties
 Syntax coloring
 ---------------
 
+.. versionchanged:: 3.1
+
+  Added support for colored terminal output via colorama_.
+
 The ``django-admin`` / ``manage.py`` commands will use pretty
 color-coded output if your terminal supports ANSI-colored output. It
 won't use the color codes if you're piping the command's output to
 another program unless the :option:`--force-color` option is used.
 
-Under Windows, the native console doesn't support ANSI escape sequences so by
-default there is no color output. But you can install the `ANSICON`_
-third-party tool, the Django commands will detect its presence and will make
-use of its services to color output just like on Unix-based platforms.
+Under Windows, the legacy ``CMD.exe`` native console doesn't support ANSI
+escape sequences so by default there is no color output. But there are three
+ways to get color output on Windows from Django:
+
+* Install `ANSICON`_, a third-party tool that allows ``CMD.exe`` to process
+  ANSI color codes. Django commands will detect its presence and will make use
+  of its services to color output just like on Unix-based platforms.
+* Use a newer version of PowerShell or `Windows Terminal
+  <https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701>`_
+  that has built-in support for ANSI color codes, and "fake" the installation
+  of ``ANSICON`` by setting the appropriate Environmental Variable
+  (``ANSICON="on"``). The easiest way to do this may be to add to following
```

Comment:
I would drop this example. 

(Surely just `set ANSICON=on` is the easiest way to set an environment variable...)

Reply:
> (Surely just set ANSICON=on is the easiest way to set an environment variable...)

If only...

On Powershell `set ANSICON "on"` will indeed set a variable, but it is within the Powershell session, and not the environmental variable (so it's not going to be available to Django).

You could use `setx ANSICON "on"`, which will set the environmental variable, but this is actually adding an entry to the Registry, and so won't take effect until you re-open your terminal.

But as there seems to be support for installing colorama by default, I'll cut most of the section, including the example.